recipes/quast/meta.yaml

```diff
@@ -1,44 +1,51 @@
+{% set name = "quast" %}
+{% set version = "4.5" %}
+{% set hash = "873ed8cd69e2d92fb87d75a37ecb75ea" %}
```

Comment:
Name and hash don't need to be variables. Only the version, because it occurs at multiple places and changes with a new release. However, this is not crucial so you can also keep it like this if you prefer this style.